packages/react/src/components/Storage/File-Uploader/hooks/useFileUploader/__tests__/useFileUploader.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useFileUploader } from '..';
+
+// mock `aws-amplify` to prevent logging auth errors during test runs
+jest.mock('aws-amplify');
+
+describe('useFileUploader', () => {
+  beforeEach(() => {});
+
+  it('exists', () => {
+    const { result } = renderHook(useFileUploader);
+
+    expect(result).toBeTruthy();
+  });
+});
```

Comment:
I believe we can add more unit tests regarding its functionality, e.g. using `setInDropZone`, handling drags, etc.

Reply:
OK, I'll add those to a different PR

Reply:
Not sure how to test drag and drop properly in an unit test. Maybe it's better for E2e?

Reply:
I think just firing drag event (maybe `dom-testing-library` has something helpful?) and seeing if `event.stopPropagation()` is called or `inDropZone` is updated as expected.

Not blocking though, as you mentioned we can add it to another PR